examples/structured_outputs/structured-outputs.ts

```diff
@@ -0,0 +1,70 @@
+import ollama from 'ollama';
+import { z } from 'zod';
```

Comment:
```suggestion

import { z } from 'zod';
```
nit, to match the first example